cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -178,7 +178,7 @@ <h3 class="o-post-preview_title">
             {% endfor %}
             </div>
 
-            {% if post.tags.exists() %}
+            {% if post.tags.exists() and controls.categories.page_type != 'foia-freq-req-record' %}
```

Comment:
This breaks when I visit http://localhost:8000/policy-compliance/amicus/. It gives this error message: `'None' has no attribute 'categories'`. Do some lists not get a 'controls' variable?

Reply:
😭

Reply:
OOooo it looks to me that amicus uses the post preview, but is not a filterable list. Of course! We need to be careful with post preview edits because it's just something that's consumed by filterable lists, but they are independent things.